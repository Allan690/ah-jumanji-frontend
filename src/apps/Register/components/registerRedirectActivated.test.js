import React from 'react';
import {shallow} from 'enzyme';

import RegisterRedirectActivated from "./registerRedirectActivated";


describe('<RegisterRedirectActivated />', () => {
 const mockRegisterRedirectActivatedFn = jest.fn;
 let wrapper;

 beforeEach(() => {
    wrapper = shallow(<RegisterRedirectActivated registerRedirectActivated={mockRegisterRedirectActivatedFn} />);
  });

  it('should match the snapshoot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render register redirect activated page correctly', () => {
    expect(wrapper.find('.row').exists()).toBe(true);
    expect(wrapper.find('.register-redirect-modal').exists()).toBe(true);
    expect(wrapper.find('p').text()).toBe('Account Activated!');
    expect(wrapper.find('p').length).toEqual(1);
    expect(wrapper.find('a').text()).toBe('You can now login here');
  });
});
