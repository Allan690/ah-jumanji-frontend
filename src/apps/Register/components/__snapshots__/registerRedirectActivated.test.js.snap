// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterRedirectActivated /> should match the snapshoot 1`] = `
<div
  className="row"
>
  <div
    className="bg-circle-1 bg-circle"
  />
  <div
    className="bg-circle-2 bg-circle"
  />
  <div
    className="bg-circle-3 bg-circle"
  />
  <div
    className="register-redirect-modal"
  >
    <img
      alt="success-icon"
      id="success-icon"
      src={null}
    />
    <p>
      Account Activated!
    </p>
    <a
      href="/"
    >
      You can now login here
    </a>
  </div>
</div>
`;
